app/views/users/_form.html.haml

```diff
@@ -0,0 +1,9 @@
+= simple_form_for(@user, url: url) do |f|
```

Comment:
If you want, you can call `simple_form_for [current_facility, @user] do |f|` and it'll be smart enough to point you to the right endpoint in both the new and edit cases. Then you don't need to pass in the URL.